packages/diracx-web/src/app/auth/page.tsx

```diff
@@ -0,0 +1,6 @@
+"use client";
+import { LoginForm } from "@dirac-grid/diracx-web-components/components";
```

Comment:
So if I understand correctly, if the library is used in parallel of `diracx-web`, then the local version is used. Else, the library is fetched from npm. Am I right?
Then, this mechanism should be explained somewhere (in a `docs` directory for instance).

Reply:
Yes, you understood it correctly. When the library is used in parallel with diracx-web (in a npm workspace), the local version is utilized. Otherwise, the library is fetched from npm.
I’ll make sure to add an explanation.